SilKit/source/config/ParticipantConfiguration_Full.json

```diff
@@ -219,6 +219,16 @@
       "path/to/extensions2"
     ]
   },
+  "Includes": {
```

Comment:
Is this correct to change these files here? They are used within our test suite

Reply:
You're right, I reverted this (and in the corresponding yaml)